cpp/src/arrow/compute/kernels/vector_cumulative_ops_test.cc

```diff
@@ -0,0 +1,183 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include <gtest/gtest.h>
+
+#include "arrow/array.h"
+#include "arrow/chunked_array.h"
+#include "arrow/testing/gtest_util.h"
+#include "arrow/testing/util.h"
+#include "arrow/type.h"
+
+#include "arrow/compute/api.h"
+#include "arrow/compute/kernels/test_util.h"
+
+namespace arrow {
+namespace compute {
+
+class TestCumulativeOp : public ::testing::Test {
+ protected:
+  std::shared_ptr<Array> array(std::shared_ptr<DataType>& type,
+                               const std::string& values) {
+    return ArrayFromJSON(type, values);
+  }
+
+  std::shared_ptr<ChunkedArray> chunked_array(std::shared_ptr<DataType>& type,
+                                              const std::vector<std::string>& values) {
+    return ChunkedArrayFromJSON(type, values);
+  }
```

Comment:
So for the record I don't think that such aliases are useful, and they also make reading the tests more confusing ("is `array` doing something different from `ArrayFromJSON`? I'll have to check the definition...")